docs/porting/libraries.md

```diff
@@ -1,5 +1,304 @@
+
+### Dealing primarily with the compiler
+
+This approach may be the best for small projects or projects which don't use many .NET Framework APIs.  The approach is very simple:
+
+1. Run API Port on your project.
+2. Take a quick glance and perhaps use the report as a reference for future errors.
+3. Copy all of your code over into a new .NET Core project.
+4. Work out compiler errors until it compiles, referring to the portability report if needed.
+5. Repeat as needed.
+
+Although this approach is very unstructured, the code-focused approach can lead to resolving any issues quickly, and may be the best approach for smaller projects or libraries.  A project that contains only data models may be an ideal candidate here.
+
+### Staying on .NET Framework until portability issues are resolved
+
+This approach may be the best if you prefer to have code that compiles during the entire process.  The approach is as follows:
+
+1. Run API Port on a project.
+2. Address issues by using different APIs which are portable.
+3. Keep note of any areas where you can't use a direct alternative.
+4. Repeat steps 1-3 for all projects you're porting until you're confident each is ready to be copied over into a .NET Core project.
+5. Copy the code into a new .NET Core projects.
+6. Work out any issues that you've kept note of.
+
+This careful approach is more structured than simply working out compiler errors, but it is still relatively code-focused and has the benefit of always having code that can compile.  The way you resolve certain issues that couldn't be addressed by just using another API can vary greatly.  You may find that you need to develop a more comprehensive plan for certain projects, which is covered as the next approach.
+
+### Developing a comprehensive plan of attack
+
+This approach may be best for larger and more complex projects, where restructuring of code or rewriting certain areas may be necessary to support .NET Core.  The approach is as follows:
+
+1. Run API Port on a project.
+2. Understand where in your code each non-portable type is being used and how that affects overall portability.
+
+   a.  Understand the nature of those types.  Are they small in number, but used frequently?  Are they large in number, but used infrequently?  Is their use concentrated, or is it spread throughout your code?
+   
+   b. Is it easy to isolate code that isn't portable so you can deal with it more easily?
+   
+   c. Would you need to refactor your code?
+   
+   d. For those types which aren't portable, are there alternative APIs that accomplish the same task?  For example, if you're using the `WebClient` class, you may be able to use the `HttpClient` class instead.
```

Comment:
Also because we'll eventually be moving off of github pages, I'd have to go back and change the link again.  Probably better to just go through articles and add links to types once we arrive to something more stable.